replies/goodbye.go

```diff
@@ -0,0 +1,77 @@
+package replies
+
+import "github.com/emseers/eelbot"
+
+var (
+	goodbyePrefixes = []string{
+		"gnight",
+		"g'night",
+		"gnite",
+		"g'nite",
+		"good night",
+		"goodnight",
+		"good nite",
+		"goodnite",
+		"gud night",
+		"gudnight",
+		"gud nite",
+		"gudnite",
+		"bye",
+		"good bye",
+		"goodbye",
+		"gbye",
+		"g'bye",
```

Comment:
For completeness sake we should add "Guten Morgen", German for good morning to pair with auf Wiedersehen

Reply:
Done